C22_Physics/SteveManager.h

```diff
@@ -0,0 +1,67 @@
+/*----------------------------------------------
+Programmer: Ruben Young (ray6190@rit.edu)
```

Comment:
PLS make sure to have a `deleteEntities(vector of entity pointers)`  which I can call from my LaneGrid